plugins/bazaar-backend/src/service/standaloneServer.ts

```diff
@@ -54,6 +55,7 @@ export async function startStandaloneServer(
     logger,
     database: { getClient: async () => db },
     config: config,
+    identity: {} as DefaultIdentityClient,
```

Comment:
DefaultIdentityClient is an instance - you could change this to `IdentityApi`, or making an actual instance, or making a mock instance like `{ getIdentity: async () => undefined }` or so. I'd actually be surprised if this doesn't crash after running `yarn start` in the plugin folder?

Reply:
The stand alone server still works. It's just just used a typecast so the typing should not be there at run time?

Reply:
Sure it does :) Just a bit unexpected to see a concrete class as the typecast, rather than the interface. No big deal.